### DISCUSSION TITLE
Blynk custom/Local server

### DISCUSSION THREAD

**Discussion by fmb787** [universam1/iSpindel#597] [Ideas]
Hi,

good day..

As everyone knows (blynk ) cloud server "blynk-cloud.com" will be shut down after 2 days 

and i have blynk local server , i hope if there is option where we can add custom server IP and port 

Thanks and best regards 

![blynk server](https://user-images.githubusercontent.com/76410431/209881202-35faa391-a650-4be3-9057-89e5f8bba1ed.png)

**Comment by fmb787** [comment]
after more than i year , can someone help

**Reply by thegreatgunbantoad** [reply]
The server is hardcoded in line 21 of https://github.com/universam1/iSpindel/blob/master/pio/lib/Sender/Sender.cpp
`#define BLYNKSERVER "blynk-cloud.com"`
Perhaps just rewriting that line and rebuilding would do for for a quick and dirty fix.  The code doesn't currently seem to use a specific port.

Obviously a better option is to take in an input on the setup screen, if just changing that server address would work I could have a look at adding an input field for it.
